.github/workflows/stimulus.yml

```diff
@@ -0,0 +1,24 @@
+name: Stimulus
+on: pull_request
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      #- name: Setup Python
+      #  uses: actions/setup-python@v2
+      - name: Configure igraph
+        run: |
+          mkdir build && cd build
+          cmake ..
+      - name: Run verification
+        run: |
+          cd 
+          python3 -m .venv venv
+          .venv/bin/pip install 'https://github.com/igraph/stimulus.git@0.8.0#egg=stimulus'
+          .venv/bin/stimulus -f functions.yaml -t types.yaml -l ci:validate -o test.cpp
+          c++ -std=c++14 -o test test.cpp -I ../include -I ../build/include -ferror-limit=1
```

Comment:
For CI purposes we can probably raise the error limit; having only one error is not helpful if someone breaks multiple definitions because you will falsely anticipate that fixing this single error will fix the build.